lib/cocoapods-core/podfile/dsl.rb

```diff
@@ -1,9 +1,5 @@
 module Pod
   class Podfile
-    # The of the methods defined in this file and the order of the methods is
-    # relevant for the documentation generated on
-    # CocoaPods/cocoapods.github.com.
```

Comment:
this should just be changed to `https://github.com/CocoaPods/guides.cocoapods.org`